x-pack/platform/plugins/shared/onechat/server/services/runner/runner.ts

```diff
@@ -109,7 +111,7 @@ export const createRunner = (deps: CreateRunnerDeps): Runner => {
     },
     runAgent: (params) => {
       const { request, defaultConnectorId, ...otherParams } = params;
-      const resultStore = createResultStore(params.agentParams.conversation);
+      const resultStore = createResultStore(params.agentParams.conversation?.rounds);
```

Comment:
We need to change that "resultStore" to some kind of artifact store - or we need another way to let tools access attachments from the current conversation. This is tracked in https://github.com/elastic/search-team/issues/11698 and we will have a sub issue for that.